src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -114,7 +131,7 @@
 
         (:error-message @state) [:div {:style {:color "red"}}
                                  "FireCloud service returned error: " (:error-message @state)]
-        :else [comps/Spinner {:text "Loading configurations for import..."}])])})
+         :else [comps/Spinner {:text "Loading configurations for import..."}])])})
```

Comment:
You should not be manually formatting. Auto-format this line. If it's wrong, we'll fix your IDE.